src/Build/BackEnd/BuildManager/BuildManager.cs

```diff
@@ -486,7 +488,7 @@ public void BeginBuild(BuildParameters parameters)
             ILoggingService InitializeLoggingService()
             {
                 ILoggingService loggingService = CreateLoggingService(
-                    _buildParameters.Loggers,
+                    (_buildParameters.Loggers ?? Enumerable.Empty<ILogger>()).Concat(GetDebuggingLoggers()),
```

Comment:
Why do you need to concat something to empty as opposed to just taking GetDebuggingLoggers()?

Reply:
If buildParam.Loggers is not null, then I need to concat with the method. If it is null, I need to take just the method. This was the shortest way I found to express it.

Reply:
This is very concise but since it's just for us, I'd rather see something like
```suggestion
                    AppendDebuggingLoggers(_buildParameters.Loggers),
```

so we have neither linq nor the empty enumerables in the main path.